test/shell/exec-usergroup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -x
+
+if ! timeout 1s sudo -A true; then
+	echo "sudo disabled: not checking exec user and group"
+	exit
+fi
+
+function setup {
+	mkdir -p /tmp/mgmt/test-exec-usergroup/
+	sudo -A chown nobody:$group /tmp/mgmt/test-exec-usergroup/
```

Comment:
where does this $group variable come from? If it's the thing at the bottom, that code should be up at the top and the "global" variables should be $SOME_VAR (in caps)